backend/src/common/utils/tokenHelper.ts

```diff
@@ -2,4 +2,12 @@ import jwt from 'jsonwebtoken';
 import { secret, expiresIn } from '../../config/jwtConfig';
 import { IAuthUser } from '../models/user/IAuthUser';
 
+const msInOneDay = 864e5;
```

Comment:
it's too hard

Reply:
https://stackoverflow.com/questions/10759647/how-to-calculate-the-date-30-days-from-today/10759688